packages/docs-reanimated/docs/reanimated-babel-plugin/about.md

```diff
@@ -0,0 +1,217 @@
+---
+id: about
+title: 'About'
+sidebar_label: 'About'
+---
+
+# Reanimated Babel Plugin
+
+This article will explain what exactly is Reanimated Babel Plugin, what are its features and limitations, without going into unnecessary details.
+
+## What is Reanimated Babel Plugin?
+
+Reanimated Babel Plugin is the part of Reanimated that allows you to employ the second JavaScript runtime created by Reanimated, called the UI/Worklet runtime. The plugin transforms your code, injects all the necessary data into it, so that the Worklet runtime can execute the code. This data comes from _worklets_. The process of worklet transformation is called _workletization_.
+
+## What is a worklet?
+
+A worklet is:
+
+- A function that contains a `'worklet'` directive at its very top, i.e.:
+
+```ts
+function foo() {
+  'worklet';
+  console.log('Hello from worklet');
+}
+```
+
+- A function that is _autoworkletizable_, i.e.:
+
+```ts
+useAnimatedStyle(() => {
+  // This function will be ran on the Worklet Runtime,
+  // hence it's in a workletizable context and will be
+  // autoworkletized. You don't need to add the 'worklet' directive here.
+  return {
+    width: 100,
+  };
+});
+```
+
+## What can be a worklet?
+
+Currently, Reanimated Babel Plugin supports the following constructs as worklets:
+
+- Arrow Function Expressions
+
+```ts
+const foo = () => {
+  'worklet';
+  console.log('Hello from ArrowFunctionExpression');
+};
+```
+
+- Function Declarations
+
+```ts
+function foo() {
+  'worklet';
+  console.log('Hello from FunctionDeclaration');
+}
+```
+
+- Function Expressions
+
+```ts
+const foo = function () {
+  'worklet';
+  console.log('Hello from FunctionExpression');
+};
+```
+
+- Object Methods
+
+```ts
+const obj = {
+  foo() {
+    'worklet';
+    console.log('Hello from ObjectMethod');
+  },
+};
+```
+
+## Autoworkletization
+
+To reduce boilerplate code and relieve the user from knowing the internals of Reanimated, Reanimated Babel Plugin tries to detect automatically whether a function should be workletized. For example, all callbacks in Reanimated API that are ran on the Worklet runtime are autoworkletized, such as:
+
+- `useAnimatedStyle`
+- `useAnimatedProps`
+- `runOnUI`
```

Comment:
Can we have a full list here? Or a link to full list in the plugin's source code?